stress_test_trigger/cloud_ymls/stress-test-daemon.yml

```diff
@@ -0,0 +1,223 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Template to run stress-test for Jina on AWS
+Parameters:
+  EC2ImageIdParam:
+    Type: String
+    Default: ami-00831fc7c1e3ddc60
+    Description: Enter Image ID. Default is for Ubuntu 20.04
+  EC2InstanceTypeParam:
```

Comment:
Is only 1 instance ok for both chunk and doc index?
Make IP of all pods available as IP1, IP2, etc.

```yaml:
port_expose:
host:
``` in each of the pods

jina, jinad, fluentd on `cfn-init` using script
and mounting of volumes

Reply:
- `PEM` file on console for logging in : key -> val

- Name for each of the instances

- Output of the infra creation of instances should be the `EIP` of the instances.

Reply:
Write `README` for infra.